src/scripts/StoreTalismanApi.js

```diff
@@ -44,6 +42,23 @@ export default class StoreTalismanApi extends StoreRestApi {
 		}
 	}
 
+	setAxiosInterceptors() {
+		this.axiosInstance.interceptors.response.use(response => {
+			const contentType = response.headers['content-type'];
+			if (
+				contentType !== 'application/json' &&
+				response.request.responseURL.includes('auth')
+			) {
+				browser.runtime.sendMessage({
+					authError: true,
+				});
+				const error = new Error(`Authentication Error`);
+				return Promise.reject(error);
+			}
+			return response;
+		});
```

Comment:
Maybe we should alert for any non-json responses in both storages? You allow such responses here unless they are not auth responses.